app/gui/view/debug_scene/component-group/src/lib.rs

```diff
@@ -359,12 +354,14 @@ fn init(app: &Application) {
     }
 
     frp::extend! { network
-        selected_entry <- multiview.selected_entry.on_change();
-        eval selected_entry([](e) if let Some((g, e)) = e {  DEBUG!("Entry {e} from group {g:?} selected") });
+        selected <- multiview.selected.on_change();
+        eval selected([](s) match s {
+            Some((g, Selected::Header)) => DEBUG!("Header selected in group {g:?}"),
+            Some((g, Selected::Entry(id))) => DEBUG!("Entry {id} from group {g:?} selected"),
+            _ => {},
```

Comment:
debug left in the code

Reply:
This is a debug scene that uses console messages for debugging purposes. This was not introduced in this PR and is only touched during refactoring.